src/cephadm/cephadm

```diff
@@ -1521,14 +1527,14 @@ def infer_config(func):
 
 
 def _get_default_image():
-    if DEFAULT_IMAGE_IS_MASTER:
+    if DEFAULT_CEPH_IMAGE == 'docker.io/ceph/daemon-base:latest-master-devel':
```

Comment:
```suggestion
    if CEPHADM_DEVEL:
```

it feels odd that this PR creates constants, but then it doesn't use a constant for the latest-master-devel image. Maybe we rename this constant, but retain it's bool type?

Reply:
@mgfritch : Thanks for the suggestion
Added a new constant for the "ceph development image". 
I prefer not to use the "CEPHADM_DEVEL" constant suggested, because we will  need to do exactly the same comparative in order to assign a value ( then it is not anymore a constant), or we need to remember to change the constant every time. ... i think more simpler in this way ( and probably , imho, more readable)

In this way we should only be worried about to change the DEFAULT constants when we move to downstream